apps/src/StudioApp.js

```diff
@@ -828,10 +828,14 @@ StudioApp.prototype.displayFeedback = function(options) {
 };
 
 /**
- *
+ * Runs the tests and returns results.
+ * @param {boolean} levelComplete Was the level completed successfully?
+ * @param {Object} options
+ * @return {number} The appropriate property of TestResults.
  */
 StudioApp.prototype.getTestResults = function(levelComplete, options) {
-  return this.feedback_.getTestResults(levelComplete, options);
+  return this.feedback_.getTestResults(levelComplete,
+      this.checkForEmptyBlocks_, options);
```

Comment:
should we pass this through as an option instead? If not, we should probably add an @param tag for it